src/utils.ts

```diff
@@ -120,7 +120,10 @@ export function timeout(delay, msg) {
   })
 }
 
-export async function measureTime(operation: Promise<any>): Promise<[any, number]> {
+// FIXME: This function is not used anywhere. Delete?
```

Comment:
This can be deleted

Reply:
Isn't there a linting option to flag dead code? Might be good to enforce that we don't have any unused code on `main` branch.

Reply:
I see an eslint plugin but it has not been updated in 3 years. There are other tools that can do that though. I'll research this some more.